schema/vault.json

```diff
@@ -50,6 +57,12 @@
     },
     "vaultNameOverride": {
       "type": "string"
+    },
+    "vaultDetailPageAssets": {
```

Comment:
What's gonna be in here? Image resources that will be displayed on the detail screen?

Reply:
The idea for this would be to have strings such as `YLA` for yearn lazy ape, `cozy` for vaults that can be insured with cozy.finance, `bowswap` for vaults that can be migrated using bowswap, etc. 

Basically just making it easier for the FE to implement methods/links that are related to only certain vaults.

Reply:
ok makes sense